roles/openshift_clock/tasks/main.yaml

```diff
@@ -0,0 +1,8 @@
+---
+- name: Install ntp package
+  action: "{{ ansible_pkg_mgr }} name=ntp state=present"
+  when: openshift.clock.enabled | bool and not openshift.common.is_atomic
```

Comment:
`when: openshift.clock.enabled | bool and not openshift.common.is_atomic | bool`